examples/7-Rasa_demo.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>RASA Integration Demo</title>
+
+    <!-- for mobile screens -->
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <!-- stylesheets are conveniently separated into components -->
+	<link rel="stylesheet" media="all" href="../component/styles/setup.css">
+	<link rel="stylesheet" media="all" href="../component/styles/says.css">
+	<link rel="stylesheet" media="all" href="../component/styles/reply.css">
+	<link rel="stylesheet" media="all" href="../component/styles/typing.css">
+	<link rel="stylesheet" media="all" href="../component/styles/input.css">
+	<style>
+	body {
+		background: #dcdde0;
+	}
+	.bubble-container {
+		height: 100vh;
+	}
+	.bubble-container .input-wrap textarea {
+		margin: 0;
+		width: calc(100% - 30px);
+	}
+	</style>
+</head>
+
+<body>
+    <!-- container element for chat window -->
+    <div id="chat"></div>
+
+    <!-- import the JavaScript file -->
+    <script src="../component/Bubbles.js"></script>
+    
+    <script>
+        // initialize by constructing a named function...
+        // ...and add text processing plugin:
+        function makeid() {
+            var result           = '';
+            var characters       = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+            var charactersLength = characters.length;
+            for ( var i = 0; i < 20; i++ ) {
+                result += characters.charAt(Math.floor(Math.random() * charactersLength));
+            }
+            return result;
+        }
+
+        //the port at which RASA is running
+        port = '1001'
+
+        //A unique client id so that the data soesn't get mismatched
+        const client_id = makeid()
+
+        console.log(client_id)
+
+        var chatWindow = new Bubbles(document.getElementById("chat"), "chatWindow", {
+
+            // the one that we care about is inputCallbackFn()
+            // this function returns an object with some data that we can process from user input
+            // and understand the context of it
+
+            // this is an example function that matches the text user typed to one of the answer bubbles
+            inputCallbackFn: function(chatObject){
+                
+                var miss = function(text_inp = null){
+                    let xhr = new XMLHttpRequest()
+                    let url = "http://localhost:"+port+"/webhooks/rest/webhook"
```

Comment:
Looks like this example requires running another server. I think we should explain how to start this server so that the example can be useful to all developers.